webnn/resources/test_data/squeeze.json

```diff
@@ -0,0 +1,679 @@
+          ],
+          "type": "float32"
+        }
+      },
+      "options": {
+        "axes": [1, 2]
+      },
+      "expected": {
+        "name": "output",
+        "shape": [2, 12, 1],
+        "data": [
+          1.1293666362762451,
+          -93.29325866699219,
+          -19.999887466430664,
+          41.11921310424805,
+          92.69351959228516,
+          -41.519378662109375,
+          -56.752288818359375,
+          11.148094177246094,
+          5.575400352478027,
+          15.709176063537598,
+          62.550445556640625,
+          -5.299276351928711,
+          -18.615251541137695,
+          -66.94721984863281,
+          94.97520446777344,
+          -21.940048217773438,
+          30.047636032104492,
+          13.805134773254395,
+          -45.086708068847656,
+          76.903564453125,
+          46.969749450683594,
+          -86.58126831054688,
+          31.64584732055664,
+          -67.96637725830078
+        ],
+        "type": "float32"
+      }
+    },
+    {
+      "name": "squeeze float32 5D tensor by eliminating four dimensions options.axes",
+      "inputs": {
+        "input": {
+          "shape": [1, 1, 1, 24, 1],
+          "data": [
+            1.1293665986622017,
+            -93.29325783541842,
+            -19.999887095191227,
+            41.119214363429876,
+            92.69352298075461,
+            -41.51937878962313,
+            -56.75228730395734,
+            11.148093991767155,
+            5.57540042674151,
+            15.709176425437747,
+            62.550443705517665,
+            -5.299276370816216,
+            -18.61525168224469,
+            -66.94721623021692,
+            94.97520289870027,
+            -21.94004801967297,
+            30.047636856615554,
+            13.805134987122344,
+            -45.08670702769,
+            76.90356276507339,
+            46.96974979938284,
+            -86.58127045567055,
+            31.645846897982068,
+            -67.96637618240129
+          ],
+          "type": "float32"
+        }
+      },
+      "options": {
+        "axes": [0, 1, 2, 4]
+      },
+      "expected": {
+        "name": "output",
+        "shape": [24],
+        "data": [
+          1.1293666362762451,
+          -93.29325866699219,
+          -19.999887466430664,
+          41.11921310424805,
+          92.69351959228516,
+          -41.519378662109375,
+          -56.752288818359375,
+          11.148094177246094,
+          5.575400352478027,
+          15.709176063537598,
+          62.550445556640625,
+          -5.299276351928711,
+          -18.615251541137695,
+          -66.94721984863281,
+          94.97520446777344,
+          -21.940048217773438,
+          30.047636032104492,
+          13.805134773254395,
+          -45.086708068847656,
+          76.903564453125,
+          46.969749450683594,
+          -86.58126831054688,
+          31.64584732055664,
+          -67.96637725830078
+        ],
+        "type": "float32"
+      }
+    }
```

Comment:
We should test the corner case of all 1's, which yields a single scalar value.

`"shape": [1, 1, 1, 1]` -> `[]`

Of course, this scalar will probably be filled with leading 1's before reaching backend implementations.

Reply:
Added such a test, thanks @fdwr！